cms/templates/container.html

```diff
@@ -51,6 +51,28 @@
                 outlineURL: "${outline_url | n, js_escaped_string}"
             }
         );
+        require(["js/models/xblock_info", "js/views/xblock", "js/views/utils/xblock_utils", "common/js/components/utils/view_utils"], function (XBlockInfo, XBlockView, XBlockUtils, ViewUtils) {
```

Comment:
@OmarIthawi Chance of putting this on multiple lines for readability?

Reply:
Thanks @johnbaldwin. It makes sense if it's our own code, but it's mostly copy and paste from edX's templates. Keeping it as is would keep the sync easy.